@Arishamays1 To fix a failing test because of this error:
```
    sodium_lib_restore = ristretto.sodium._lib # pylint: disable=protected-access
E   AttributeError: 'NoneType' object has no attribute '_lib'
```
I've added the missing `libsodium` to `host` and `run`. See the reason https://github.com/nthparty/oblivious/blob/03a59af6188b9d30dc8eab7fb0aab7ad483ba266/src/oblivious/ristretto.py#L500-L503

--- a/abs.yaml
+++ b/abs.yaml
@@ -1,5 +1,2 @@
 upload_channels:
   - sfe1ed40
-
-channels:
-  - sfe1ed40

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -24,10 +24,10 @@ requirements:
   run:
     - python
     - libsodium >=1.0.18,<1.0.19.0a0
-    - parts >=1.6
-    - fe25519 >=1.4
-    - ge25519 >=1.4
-    - bn254 >=0.1
+    - parts >=1.6,<2
+    - fe25519 >=1.4,<2
+    - ge25519 >=1.4,<2
+    - bn254 >=0.1,<0.2
 
 test:
   imports:

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -24,10 +24,10 @@ requirements:
   run:
     - python
     - libsodium >=1.0.18,<1.0.19.0a0
-    - parts >=1.6,<2
-    - fe25519 >=1.4,<2
-    - ge25519 >=1.4,<2
-    - bn254 >=0.1,<0.2
+    - parts >=1.6
+    - fe25519 >=1.4
+    - ge25519 >=1.4
+    - bn254 >=0.1
 
 test:
   imports:

